compiler/damlc/tests/src/DA/Test/ShakeIdeClient.hs

```diff
@@ -864,20 +870,87 @@ visualDamlTests = Tasty.testGroup "Visual Tests"
                 , "  where"
                 , "    signatory owner"
                 , "    controller owner can"
-                , "      nonconsuming ReducedCoin : ContractId Coin"
+                , "      nonconsuming ReducedCoin : ()"
                 , "        with otherCoin : ContractId Coin"
                 , "        do "
-                , "        cn <- fetch otherCoin "
-                , "        create this with amount = cn.amount - 10"
+                , "        cn <- fetch otherCoin"
+                , "        archive otherCoin"
                 ]
             setFilesOfInterest [fetchTest]
             expectNoErrors
             expectedTemplatePoperties fetchTest $ Set.fromList
-                [TemplateProp (Set.fromList
+                [TemplateProp "Coin"
+                    (Set.fromList
                     [   ExpectedChoices "Archive" True,
                         ExpectedChoices "ReducedCoin" False
                     ])
-                    1
+                    (Set.fromList [Exercise "F:Coin" "Archive"])
```

Comment:
The expectation should say which choice is expected to exercise the `Archive` choice of `F:Coin`.

Reply:
Got rid of that action we this test intent was to make sure fetch did not add a edge